src/gui/src/viewer/zoom-window.cpp

```diff
@@ -1134,6 +1189,7 @@ void ZoomWindow::showEvent(QShowEvent *e)
 {
 	Q_UNUSED(e)
 
+	if (m_settings->value("actionButtons").toInt() == ButtonVisibility::All) resizeButtons();
```

Comment:
There may be a better place for this one. It's needed because resizeEvent() is apparently not called when the viewer is first opened for an image.